client/src/components/QueryComponent/QueryComponent.js

```diff
@@ -244,8 +245,8 @@ class QueryComponent extends Component {
               <DefaultButton className="query-button" onClick={this.executeQuery} ariaLive="assertive">
                 {this.props.t("queryComponent.defaultButton")}
               </DefaultButton>
-              <IconButton className="query-save-button"
-                iconProps={{ iconName: this.props.t("queryComponent.iconButton"), style: { color: "black" } }}
```

Comment:
Does this localized value need to be removed from the localization files? Or is it used in other places?